lib/service/navigation_service.dart

```diff
@@ -1043,4 +1043,11 @@ class NavigationService {
       isDismissible: true,
     );
   }
+
+  Future<void> openUrl(Uri uri) async {
+    if (navigatorKey.currentContext != null &&
```

Comment:
are you sure that we need this check?
i haven't test that case but custom-taps isn't app context dependence

Reply:
removed check